src/splitInteger.test.js

```diff
@@ -3,10 +3,18 @@
 describe(`Function 'splitInteger':`, () => {
   const splitInteger = require('./splitInteger');
 
-  it(`should be declared`, () => {
+  it(`function 'splitInteger' should be declared`, () => {
```

Comment:
You don't need the words `function 'splitInteger'`, you already have them in the `describe` section.